basket/petition/tests/test_views.py

```diff
@@ -54,7 +55,7 @@ def test_petition_post_success(client, mocker):
 
 @pytest.mark.django_db
 def test_petition_post_invalid(client, mocker):
-    mock_send_mail = mocker.patch("basket.petition.views.send_email_confirmation")
+    mock_send_mail = mocker.patch("basket.petition.models.send_email_confirmation")
```

Comment:
I was expecting to see `"basket.petition.models.Petition.send_email_confirmation"` - is that patching def right?

Reply:
Ah, I'm patching the task which the model method calls.